src/components/app_catalog/catalogs/index.js

```diff
@@ -54,12 +54,6 @@ class Catalogs extends React.Component {
               </div>
             </React.Fragment>
           )}
-          <p className='well'>
```

Comment:
This is not on topic for this PR sorry. It's something I forgot to remove before enabling app installation.